lib/std/http/method.zig

```diff
@@ -14,4 +14,87 @@ pub const Method = enum {
     OPTIONS,
     TRACE,
     PATCH,
+
+    /// Returns true if a request of this method is allowed to have a body
+    /// Returns null if it also depends on other conditions
```

Comment:
It always depends on other conditions. even a POST might have no body.

Reply:
its about *shouldn't*, something the client/server implementation can use in debug mode, the info is from MDN

Reply:
gonna rework the doc comment to reflect this